db/migrate/20200616175358_change_incident_report_boolean_defaults.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeIncidentReportBooleanDefaults < ActiveRecord::Migration[5.2]
+  def change
+    %i[
+      incident_involved_a_van
+      assistance_requested
+      chair_on_lift
+      lift_deployed
+    ].each do |column_name|
+      change_column_null :incident_reports, column_name, true
+    end
+  end
```

Comment:
Why make it so these columns can be null but also make the default false? Like nothing seems harmful here, it just seems like if you made the default false for the columns then you wouldn't also have to allow them to be null.

Reply:
null didn't need to be specified in the schema so I removed it.

Reply:
I still don't understand I'm sorry. I guess I'm not getting the use of the word "specified" here?

Reply:
If I never set null: false when I originally created the columns then set the default as false then the result would be the same.

Reply:
But I feel like if you don't run this migration the result would be the same.

Reply:
I could be wrong, but I just want to understand. If it can't be null but the default is false that's OK because we don't want it to be null.
Unless you're thinking old data would still be null and you want them not to violate database constraints or something?

Reply:
Yeah I see what you mean.

Reply:
So, after trying to set up incidents again I get `Mysql2::Error: Field 'assistance_requested' doesn't have a default value` when trying to seed. This could be fixed by **either** allowing null or setting a default. Well really you could do both but if you don't want to allow null just set default to false.